t/09_qesapdeployment.t

```diff
@@ -559,7 +559,7 @@ subtest '[qesap_file_find_string] fail' => sub {
     # grep return 1 in case of string NOT matching
     $qesap->redefine(script_run => sub { push @calls, $_[0]; return 1; });
 
-    my $res = qesap_file_find_string(file => 'JACQUES', search_string => 'Timed out waiting for last boot time check');
+    my $res = qesap_file_find_strings(file => 'JACQUES', search_strings => ['Timed out waiting for last boot time check']);
 
     note("\n  -->  " . join("\n  -->  ", @calls));
     ok $res == 0, 'Return is 0 when string is not detected';
```

Comment:
here you should check that function is calling grep 3 times exactly

Reply:
Why 3? The call only provides a single string, so wouldn't it be called just once?